.github/workflows/release.yaml

```diff
@@ -265,8 +296,24 @@ jobs:
           check_artifacts: true
           name: npm-package
 
-      - uses: softprops/action-gh-release@v1
+      - name: Decompress npm package
+        run: tar -xzf package.tar.gz
+
+      - name: Modify version
+        run: |
+          echo "Updating version in root package.json"
+          npm version --prefix release ${{ inputs.version || github.ref_name }}
+
+          echo "Updating version in lib/vscode/product.json"
+          tmp=$(mktemp) 
+          jq '.codeServerVersion = "${{ inputs.version || github.ref_name }}"' release/lib/vscode/product.json > "$tmp" && mv "$tmp" release/lib/vscode/product.json
```

Comment:
I think we need to strip the `v` on these.

Reply:
Nice catch!